packages/broker/src/config/ConfigWizard.ts

```diff
@@ -1,254 +1,544 @@
-            default: defaultPort
+            theGraphUrl,
         }
     })
+}
 
-    return [
-        selectApiPluginsPrompt,
-        ...portPrompts
-    ]
+async function getNativeBalance(
+    network: 'polygon' | 'mumbai',
+    address: string
+): Promise<BigNumber> {
+    const url = config[network].rpcEndpoints[0]?.url || ''
+
+    if (!/^https?:/i.test(url)) {
+        throw new Error('Invalid RPC')
+    }
+
+    return new providers.JsonRpcProvider(url).getBalance(address)
 }
 
-export const PROMPTS = {
-    privateKey: PRIVATE_KEY_PROMPTS,
-    plugins: createPluginPrompts(),
-}
-
-export const storagePathPrompts = [{
-    type: 'input',
-    name: 'storagePath',
-    message: 'Select a path to store the generated config in',
-    default: getDefaultFile()
-},
-{
-    type: 'confirm',
-    name: 'overwrite',
-    message: (answers: inquirer.Answers): string => `The selected destination ${answers.storagePath} already exists, do you want to overwrite it?`,
-    default: false,
-    when: (answers: inquirer.Answers): boolean => existsSync(answers.storagePath)
-}]
-
-export const getConfig = (privateKey: string, pluginsAnswers: PluginAnswers): any => {
-    const config = { ... CONFIG_TEMPLATE, plugins: { ... CONFIG_TEMPLATE.plugins } }
-    config.client.auth.privateKey = privateKey
-
-    const pluginKeys = Object.keys(PLUGIN_NAMES)
-    pluginKeys.forEach((pluginKey) => {
-        const pluginName = PLUGIN_NAMES[pluginKey]
-        const defaultPort = DEFAULT_CONFIG_PORTS[pluginKey]
-        if (pluginsAnswers.enabledApiPlugins && pluginsAnswers.enabledApiPlugins.includes(pluginName)) {
-            let pluginConfig = {}
-            const portNumber = parseInt(pluginsAnswers[`${pluginName}Port`])
-            if (portNumber !== defaultPort) {
-                const portObject = { port: portNumber }
-                if (pluginName === PLUGIN_NAMES.HTTP) {
-                    // the http plugin is special, it needs to be added to the config after the other plugins
-                    config.httpServer = portObject
-                } else {
-                    // user provided a custom value, fill in
-                    pluginConfig = portObject
-                }
-            }
-            config.plugins![pluginName] = pluginConfig
-        }
+async function getOperatorNodeAddresses(
+    network: 'polygon' | 'mumbai',
+    operatorAddress: string
+): Promise<null | string[]> {
+    const url = config[network].theGraphUrl
+
+    const resp = await fetch(url, {
+        method: 'POST',
+        body: JSON.stringify({
+            query: `query { operator(id: "${operatorAddress}") { nodes } }`,
+        }),
     })
 
-    return config
-}
+    const { data } = z
+        .object({
+            data: z.object({
+                operator: z.union([
+                    z.null(),
+                    z.object({ nodes: z.array(z.string()) }),
+                ]),
+            }),
+        })
+        .parse(await resp.json())
```

Comment:
Does this fix some issue with `inquirer/prompts` library?

Reply:
What issues?

Reply:
Ah time line numbers have changed. Maybe referred the `progress` function.